jupyterlab/handlers/extension_manager_handler.py

```diff
@@ -87,7 +90,28 @@ def list_extensions(self):
         build_check_info = _build_check_info(app_options)
         _ensure_compat_errors(info, app_options)
         extensions = []
-        # TODO: Ensure loops can run in parallel
+
+        # TODO: the three for-loops below can be run concurrently
+        for name, data in info['federated_extensions'].items():
+            status = 'ok'
+            pkg_info = data #yield self._get_pkg_info(name, data)
+            if info['compat_errors'].get(name, None):
+                status = 'error'
+            extensions.append(_make_extension_entry(
+                name=name,
+                description=pkg_info.get('description', ''),
+                url=data.get('url', ''),
+                enabled=(name not in info['disabled']),
+                core=False,
+                # Use wanted version to ensure we limit ourselves
+                # within semver restrictions
+                latest_version=data['version'],
+                installed_version=data['version'],
+                status=status,
+                install=data['install'],
```

Comment:
@echarles - do you want to change this to something like the following to I think take care of the error you are seeing?

```suggestion
                install=data.get('install', {}),
```